test/functional/dispatch-handle-run.c

```diff
@@ -270,17 +269,17 @@ void functional_dispatch_handle_run(UNUSED(void **state))
 {
   connection_request_event_info info;
   struct api_error err = ERROR_INIT;
-  string key, functionname, invalidfunctionname;
+  string functionname, invalidfunctionname;
 
   register_test_function();
 
   info.api_error = err;
   assert_false(info.api_error.isset);
   info.con = MALLOC(struct connection);
   info.con->closed = true;
+  memcpy(info.con->cc.pluginkeystring, pluginkey, PLUGINKEY_STRING_SIZE);
```

Comment:
The pluginkeystring is an array and not of type `char *`. So copying seems legit and necessary.